reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClientOperations.java

```diff
@@ -275,6 +275,9 @@ protected void onInboundCancel() {
 			return;
 		}
 		//"FutureReturnValueIgnored" this is deliberate
+		if (log.isDebugEnabled()) {
+			log.debug(format(channel(), "Cancelling inbound stream, closing channel."));
```

Comment:
Maybe "cancelled" (passive) better reflects that something external got this stream cancelled, vs "cancelling" (active) which leaves it more unclear, did we cancel or did something else?

```suggestion
			log.debug(format(channel(), "Inbound stream cancelled, closing channel."));
```

Reply:
indeed, it seems more appropriate to use "cancelled" instead fo "cancelling", please check the commit https://github.com/reactor/reactor-netty/pull/2585/commits/57527cebed90fcfc8dd41bcbb58718be5a21b3f3

To be consistent, I have also done the same for the following classes which are already implementing the _onInboundCancel_ method and which are are already logging cancels before closing:

- WebSocketClientOperations
- WebSocketServerOperations
- QuicStreamOperartions